source/includes/_embed2.md.erb

```diff
@@ -53,12 +53,12 @@ If you still require recommendations on how best to implement the Embed2 script,
 ...
 <!-- Define the Embed script afterwards -->
 ```
-Embed2 supports a number of configurable options which can be set through an `adaSettings` object you define on your `window` scope. Alternatively, settings can also be passed to the `adaEmbed.start` method (see [Lazy Mode](#embed2-advanced-setup-lazy-mode) below).
```

Comment:
Removed 'below' since it's not needed because we're linking to the thing. This also makes our doc structure more brittle because we can't move things around easily.